sdk/monitor/monitor-opentelemetry-exporter/review/monitor-opentelemetry-exporter.api.md

```diff
@@ -20,16 +19,20 @@ export interface AzureExporterConfig {
     connectionString?: string;
 }
 
-// Warning: (ae-forgotten-export) The symbol "AzureMonitorBaseExporter" needs to be exported by the entry point index.d.ts
-//
+// @public
+export class AzureMonitorBaseExporter {
```

Comment:
since no one is supposed to instantiate it directly, should it be an abstract class?